internal/command/sandbox/apply.go

```diff
@@ -88,6 +89,17 @@ func apply(cfg *config.SandboxApply, out, log io.Writer, args []string) error {
 	fmt.Fprintf(log, "Created sandbox %q (routing key: %s) in cluster %q.\n\n",
 		req.Name, resp.RoutingKey, *req.Spec.Cluster)
 
+	if len(req.Spec.Local) > 0 && !sbmgr.IsWatcherRunning(status) {
+		// We don't really know here if it is a temporal problem or if we are
```

Comment:
do we want to add an operator version method in the tunnel api to help disambiguate cases like this?  if so, maybe add operator version to the status output?

Reply:
I can add it if we want, but I don't think it is strictly needed for this specific case.
We would have to check the version periodically in `locald`, because it may change over time.

Reply:
I think we will eventually need something like that in order to facilitate implementing compatibility in the client.  I understand it's not trivial b/c of the retries.  Not sure if you think it's worthwhile in the context of this PR, but I'd encourage it.  So, I'm ok if it's too much for this specific case, but maybe tracking adding that in an issue would help.

Reply:
I will implement this and https://github.com/signadot/cli/pull/92#issuecomment-1807949979, in a stacked PR on top of this one.

Reply:
@scott-cotton: here is the PR: https://github.com/signadot/cli/pull/95